e2e/test/scenarios/dashboard-filters/dashboard-filters-auto-apply.cy.spec.js

```diff
@@ -391,7 +391,7 @@ describe("scenarios > dashboards > filters > auto apply", () => {
         .should("be.disabled");
     });
 
-    it("should not display a toast even when a dashboard takes longer than 15s to load", () => {
+    it("should not display a toast even when a dashboard takes longer than 15s to load (metabase#28756)", () => {
```

Comment:
@WiNloSt I'm requesting review only from you since you were the author of the fix - please confirm whether my conclusion in PR description is correct.

Reply:
I could not reproduce the issue in `master`. I will have to dig deeper.